src/Miunie.InversionOfControl/Miunie.InversionOfControl.csproj

```diff
@@ -4,10 +4,24 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="../Miunie.Core/Miunie.Core.csproj" />
-    <ProjectReference Include="../Miunie.Discord/Miunie.Discord.csproj" />
-    <ProjectReference Include="../Miunie.LiteDbStorage/Miunie.LiteDbStorage.csproj" />
+    <ProjectReference Include="..\Miunie.Discord\Miunie.Discord.csproj" />
+    <ProjectReference Include="..\Miunie.Json\Miunie.Json.csproj" />
+    <ProjectReference Include="..\Miunie.LiteDbStorage\Miunie.LiteDbStorage.csproj" />
```

Comment:
You're overriding the universal `/` path identifiers with Windows specific `\`.

Let's be nice to our penguin friends. :penguin: